numpy/ma/core.py

```diff
@@ -4356,13 +4361,15 @@ def count(self, axis=None, keepdims=np._NoValue):
                     raise ValueError("'axis' entry is out of bounds")
                 return 1
             elif axis is None:
+                if kwargs.get('keepdims', False):
```

Comment:
Does not look right, as the default value of `keepdims` is `np._NoValue`, which evaluates as true. Because `keepdims` will always have a value, maybe just `keepdims is True`